src/org/salesforce/apexdoc/ApexDoc.java

```diff
@@ -208,7 +210,7 @@ public static ClassModel parseFileContents(String filePath) {
                 }
 
                 // gather up our comments
-                if (strLine.startsWith("/*")) {
+                if (strLine.startsWith("/**")) {
```

Comment:
AH, if those files are in github, I will take a look and use them to verify my changes moving forward. The larger my selection of live examples, the better. Is this what you talked about in the issue when you said you tried to fix this yourself?